specs/commandline/policy.md

```diff
@@ -67,6 +67,110 @@ The `notation policy` command provides a user-friendly way to manage trust polic
 }
 ```
 
+Policy language version 1.1 added support for verifying signatures associated with blob artifacts. User can use `scopes` field as a Policy selector string to decide which Policy gets applied to which blob. The `--policy-scope` argument provided in `notation blob verify` command will dictate which Policy gets picked from the policy configuration file and applied for verification. To get started with verifying blob signatures, users can refer to the following trust policy configuration sample. In this sample, there are three policies configured for different requirements:
+
+- The Policy named "blob-verification-policy" is for verifying blob artifacts signed by Wabbit Networks and scoped to `blob-verification-selector`.
+- Policy named "skip-blob-verification-policy" is for skipping verification on blob artifacts scoped to `skip-blob-verification-selector`.
+- Policy "wildcard-blob-verification-policy" is for auditing verification results when user wants to apply a wildcard policy by not providing `--policy-scope` argument in `notation blob verify` command.
+
+```jsonc
+{
+    "version": "1.1",
```

Comment:
This is a new version of trust policy, it should be included in the https://github.com/notaryproject/specifications right?

Reply:
It will be. I wanted to start from the UX side and work backwards. You can see the policy updates [here](https://hackmd.io/ewbJr2ZnT4a8U1ObDVXcSw?view#CLI-Spec). I will be raising a PR for policy spec updates shortly.

Reply:
I see. Then, could you create an issue in the https://github.com/notaryproject/specifications repo and link this PR there, as a reminder to add the spec.

Reply:
Good idea. https://github.com/notaryproject/specifications/issues/281